frontend/src/components/common/comment-item/comment-item.tsx

```diff
@@ -13,12 +15,23 @@ type Props = {
 };
 
 const CommentItem: React.FC<Props> = ({ comment, onClick, hasTimestamp }) => {
+  const { user } = useAppSelector(({ auth }) => ({
```

Comment:
common component shouldn't know anything about store
you need to pass it through props